cmd/selfupdate/verify.go

```diff
@@ -10,9 +10,9 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/ProtonMail/go-crypto/openpgp"
```

Comment:
How did you choose this package?

Reply:
There wasn't a whole lot of good alternatives to pick from. This was one which had recent commits, and came from a somewhat known company, with privacy and encryption related focus, though I cannot vouch for them any more then others..

The deprecation notice in https://pkg.go.dev/golang.org/x/crypto/openpgp and the linked issue there tries to steer users away from OpenPGP in general. The ProtonMail variant is mentioned at the of the issue description:

> Every major player that actually relies on OpenPGP and Go maintains a fork (https://github.com/keybase/go-crypto, https://github.com/ProtonMail/go-crypto, ...). Developers would be better served by finding those, and by centralizing efforts onto one, without blocking on the resource limits of the Go team...

If this is something that needs more consideration, maybe I should split it out in a separate PR? I'll probably be away a lot the coming weeks, so then I could try to get the rest merged..